browser/css/device-desktop.css

```diff
@@ -1,15 +1,15 @@
 /* CSS specific for desktop browsers. */
 
 /* Related to selectionMarkers.css on formulabar*/
-#tb_formulabar_item_formula .inputbar_selection_handles *{
+.inputbar_selection_handles *{
```

Comment:
we don't use tuneled formulabar - we don't have selection handles

so we can remove these 2 rules (as they even were not used for a long time)